dataactcore/config_example.yml

```diff
@@ -38,17 +38,23 @@ broker:
     # Uses "{}" because it requires string formatting to insert query parameters
     # If the URL is provided, the file name provided will be used when uploading to S3
     d1_url: https://data-usas.uat.workplace.gov/api/contracts?CGAC={}&date_signed_beginning={}&date_signed_end={}
+    d1_file_name: d1_data.csv
+    award_procurement_url: https://data-usas.uat.workplace.gov/api/contracts?CGAC={}&date_signed_beginning={}&date_signed_end={}
```

Comment:
Do we want to put the real endpoint in our sample config? I haven't used it so am unsure if there's anything preventing other people from hitting it and possibly endangering the health of the service.